leakcanary-android-core/src/main/res/values/leak_canary_colors.xml

```diff
@@ -27,6 +27,7 @@
   <color name="leak_canary_gray">#2F2F2F</color>
   <color name="leak_canary_gray_darkest">#151C1F</color>
   <color name="leak_canary_gray_darkest_50p">#c1151C1F</color>
```

Comment:
This is actually 76% alpha

Reply:
I was aiming for 75%, off by one error I guess.